client/src/components/Team/team.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from "react";
+import "./team.css"
+import GitHubIcon from '@material-ui/icons/GitHub';
+
+const Team = () => {
+    const projectadmin = {
+        name: 'Purnima Sharma',
+        title: 'Project Admin',
+        githublink: 'https://github.com/purnima143',
+        image: 'https://avatars1.githubusercontent.com/u/57852378?v=4?s=100'
+    };
+    const mentor = [
+        {
+            name: 'Ahmed Mawia',
+            title: 'Project Mentor',
+            githublink: 'https://github.com/maw1a',
+            image: 'https://avatars.githubusercontent.com/u/48566609?v=4'
+        },
+        {
+            name: 'Ishu Raj',
+            title: 'Project Mentor',
+            githublink: 'https://github.com/ir2010',
+            image: 'https://avatars.githubusercontent.com/u/46022116?v=4'
+        },
+        {
+            name: 'Vivekkumar Javiya',
+            title: 'Project Mentor',
+            githublink: 'https://github.com/codewithvk',
+            image: 'https://avatars.githubusercontent.com/u/61119120?v=4'
+        }
+    ];
+    const [contributors, setContributors] = useState([]);
+    useEffect(() => {
+        fetch("https://api.github.com/repos/purnima143/Kurakoo/contributors?per_page=1000")
+          .then(res => res.json())
+          .then(data => {
+            console.log(data);
```

Comment:
Here mentor name also repeated so remove the mentors and project admin name from the contributor.